src/app/tvshows/store/tvshows.selectors.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import { select } from '@angular-redux/store';
+import { Observable } from 'rxjs/Observable';
+import { TvShowData } from './tvshows.models';
+import { RootState } from '../../store';
+
+export class TvShowSelectors {
+  @select(getSortedResults) public readonly tvshows$: Observable<TvShowData[]>;
+  @select(getIsFetching) public readonly isFetching$: Observable<boolean>;
+}
+
+/*
+ I have to define these as functions and then pass them to @select() due to a
+ typescript bug which causes builds to fail when -aot is enabled.
+
+ I originally had these specified as arrow functions within the @selector()
+ decorator
+ */
+
+function getSortedResults(state: RootState) {
+  return state.tvshows.searchResults.sort(sortShowsByTitle);
+}
+
+function getIsFetching(state: RootState) {
+  return state.tvshows.isFetching;
+}
+
+function sortShowsByTitle(a: TvShowData, b: TvShowData) {
+    if ( a.title.toLowerCase() < b.title.toLowerCase() ) {
```

Comment:
Indentation seems off here. Do you have the `editorconfig` plugin installed in your editor? If not, I recommend doing so to help ensure consistent indentation.

Reply:
Ah, I had it installed in my vdi but not on the mac.  Will do.  I did specify correct indentation in the tslint.json, I don't know why it didn't get caught.